packages/ag-charts-website/src/content/docs/layout/index.mdoc

```diff
@@ -7,40 +7,30 @@ This section explains how the chart and its components are sized and laid out wi
 
 ## Chart Size
 
-The chart out of the box uses the minimal styling to ensure the chart is visible, and has a default minimum width and height of 300px.
+The chart will auto-size by default, taking the size of the container element and auto-sizing the chart dynamically. Use the `width` and/or `height` options if a fixed size is required.
 
-Below is an example of a chart with no CSS applied. In this case, the default for CSS is to make the width as large as possible and the height as small as possible (the minimum of 300px).
+When auto-sizing, the chart will default to a minimum width and height of 300px. Set `minHeight: 0` and/or `minWidth: 0` to remove this constraint.
 
-{% chartExampleRunner title="Chart Size" name="chart-unsized" type="generated" /%}
+### Size Changes
 
-If it is required, the minimum width and height can be adjusted with the `minWidth` and `minHeight` options.
+The chart monitors the size of the `container` element, and resizes dynamically.
 
-```js
-{
-    minWidth: 100,
-    minHeight: 100
-}
-```
+When the container size changes, the layout process is repeated for the new canvas space.
+As most chart components are a fixed size, it is usually the series-area which shrinks or grows, with the other components moving around or adjusting alignment.
 
-An explicit size can be set using the `width` and `height` options. In this case, the `minWidth` and `minHeight` are ignored.
+## Sizing the Chart Container
 
-```js
-{
-    width: 400,
-    height: 400
-}
-```
+The chart never affects the size of the `container` element.
```

Comment:
This isn't true, the `minWidth` and `minHeight` affect the container size by default for `<div>`-based containers (they put a minimum size constraint in place).